.github/workflows/ci.yml

```diff
@@ -0,0 +1,20 @@
+name: CI
+on:
+  pull_request:
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v2.1.0
+        with:
+          fetch-depth: 100
+      - name: Fetch tags
+        run: git fetch --depth=100 origin +refs/tags/*:refs/tags/*
+      - uses: olafurpg/setup-scala@v7
```

Comment:
Wouldn't it be better to use the standard scala template action provided by GitHub Actions team? I don't like having another dependency on a one-man-show project.

Reply:
That's _the de-facto official_ Scala setup (I don't know if there's any other, btw). It's not one-man-show thing, this person has worked at Scala Center and now at Twitter, so it has always been an institution-backed thing, it's just that it was always published under his account.

Reply:
I checked the project before I wrote the comment and according to contributors it looks like one-man-show project.

So what's wrong with the official Scala support for GitHub Actions? And cannot we improve this official way insted?

![image](https://user-images.githubusercontent.com/273052/91425803-ed5a2f80-e85b-11ea-962d-1d55149b4d30.png)

Reply:
I didn't even know this exists... I've always seen `olafurpg/setup-scala` being used. I'll have to have a look.

Reply:
that's really funny... when I searched for `github actions scala` or `github setup scala`, I only got links to https://github.com/olafurpg/setup-scala
I had to click on the `Actions` tab in GitHub to see the suggested setup and when I clicked on it, I saw this

![image](https://user-images.githubusercontent.com/9019485/91427014-95bcc380-e85d-11ea-9117-ded8114bd2d5.png)

so it seems that it only sets up Java (and somehow sbt is available too?)

Are you sure we want to go this way? Because I already have experience with `setup-scala` and I know it works OK (as in _doesn't get in your way and you don't need to know it exists_). It also seems to be very widely used:
 * https://github.com/scalacenter/scalafix/blob/master/.github/workflows/ci.yml#L20
 * https://github.com/scalacenter/bloop/blob/master/.github/workflows/ci.yml#L26
 * https://github.com/scalameta/scalafmt/blob/master/.github/workflows/ci.yml#L21
 * https://github.com/scalameta/metals/blob/main/.github/workflows/ci.yml#L39
 * https://github.com/typelevel/cats-effect/blob/series/2.x/.github/workflows/ci.yml#L33
 * https://github.com/typelevel/fs2/blob/main/.github/workflows/ci.yml#L37
 * https://github.com/monix/monix/blob/series/3.x/.github/workflows/build.yml#L27

I've never seen anything else, like `actions/setup-java`. Either people don't use GitHub Actions, or they go with `olafurpg/setup-scala`.

I personally don't even have any stake in this game. It's just that I see that everybody does this thing, so unless we have a very good reason not to, I suggest we do the same :D